cmd/skywire-cli/commands/node/gen-config.go

```diff
@@ -186,3 +189,19 @@ func defaultSkyproxyClientConfig() visor.AppConfig {
 		Port:      routing.Port(skyenv.SkyproxyClientPort),
 	}
 }
+
+func getLocalIPAddress() string {
```

Comment:
Probably we need to test the function somehow. It looks like it does what we need, but I'm not sure it would work correctly in all cases.

Reply:
I am pretty sure this will not work in all cases as users may have non-standard network interfaces on their machines and fairly complex setups. This is just meant as a rough measure to cover most of the standard cases automatically. If you see any big areas that we should check for though, please let me know.

Reply:
I think it's fine then.